src/Mod/Sketcher/Gui/DrawSketchHandler.h

```diff
@@ -168,6 +168,32 @@ class SketcherGuiExport DrawSketchHandler: public Gui::ToolHandler
     // get the actual highest edge index, the next use will be +1
     int getHighestCurveIndex();
 
+    // Helpers for seekAutoConstraint :

```

Comment:
Did you consider making these `protected`? Or is the intent that they do form part of the public API of this class?

Reply:
No reason for those to be public so far.